src/App.js

```diff
@@ -1,12 +1,68 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
 import './App.css';
+import Pokemon from "./components/Pokemon";
+import Button from "./components/Button"
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemonArray, setPokemonArray] = useState([]);
+    const [currentPage, setCurrentPage] = useState('https://pokeapi.co/api/v2/pokemon');
+    const [nextPage, setNextPage] = useState();
+    const [prevPage, setPrevPage] = useState();
+    const [loading, toggleLoading] = useState(false);
+    const [error, setError] = useState(false);
+
+    useEffect(() => {
+        async function fetchEmAll() {
+            toggleLoading(true);
+            setError(false);
+
+            try {
+                const result = await axios.get(currentPage);
```

Comment:
Je zou hier nog spreading kunnen toepassen
```suggestion
                const { data: { next, previous, results } } = await axios.get(currentPage);
```